oap-server/server-library/library-elasticsearch-client/src/test/java/org/apache/skywalking/library/elasticsearch/ITElasticSearchTest.java

```diff
@@ -104,6 +104,16 @@ public static Collection<Object[]> es() {
                                        "docker.elastic.co/elasticsearch/elasticsearch-oss"))
                     .withEnv("plugins.security.disabled", "true")
                     .withStartupTimeout(java.time.Duration.ofMinutes(5))
+            },
+            {
+                "OpenSearch 2.4.0",
```

Comment:
@wankai123 mentioned, we should consider to separating ElasticSearch IT from general ITs, and try to only booting ElasticSearch once per version.

Reply:
Yes, I found our ElasticSearch IT frequently fails recently and costs a lot of time.

Reply:
And IT had become the longest CI task in our e2e task list.